README.md

```diff
@@ -10,3 +10,34 @@
 [![DOI](https://zenodo.org/badge/85334761.svg)](https://zenodo.org/badge/latestdoi/85334761)
 
 IMPROVER is a library of algorithms for meteorological post-processing and verification.
+
```

Comment:
Perhaps we should also include a Python 3.7 badge above, in addition to the Python 3.6 badge, as the instructions below are for Python 3.7?

Reply:
done 👍